posthog-core/test/posthog.featureflags.spec.ts

```diff
@@ -16,6 +16,7 @@ describe('PostHog Core', () => {
     'json-payload': true,
   })
 
+  // NOTE: Aren't these supposed to be strings?
```

Comment:
Comment for the FF team that I pinged them about - our tests are pre-parsed when they should be json strings